docs/concepts.rst

```diff
@@ -947,13 +947,27 @@ be used:
 
     ...
 
-    flow = flow.reloading()
+    flow.reload()
     flow.get('my_entity')
 
-This attempts to reload all modules associated with the flow, and then return
-a re-imported version of the flow.  (This is a fairly magical procedure -- in
-complicated cases, it may not be able to figure out how to do this.  In these
-cases it will try to throw an exception rather than fail silently.)
+This attempts to reload all modules associated with the flow, and then updates
+the flow instance to use the reloaded modules.  (This is a fairly magical
+procedure -- in complicated cases, it may not be able to figure out how to do
+this.  In these cases it will try to throw an exception rather than fail
+silently.) All future operations on the flow will reflect its updated status.
+
+You can also use the :meth:`Flow.reloading <bionic.Flow.reloading>` method to
+get a new copy of the flow that uses reloaded modules. The existing
```

Comment:
nit: the second sentence might be a little tighter as ", without modifying the original flow instance."